src/qibolab/__init__.py

```diff
@@ -8,7 +8,6 @@
 
 from qibo.config import raise_error
 
-from qibolab.platform import Platform
```

Comment:
I believe this is used in qibolab_platforms_qrc when allocating new platforms. I think the safest option to avoid the circular import is to move the `ExecutionParameters` and related classes to a new file and import them here so that the interface doesn't change.

Reply:
I can do that, would it be better to do it on other PR ?

Reply:
https://github.com/qiboteam/qibolab/pull/502